src/main/java/no/nav/familie/integrasjoner/dokarkiv/metadata/BarnetrygdVedtakMetadata.kt

```diff
@@ -0,0 +1,18 @@
+package no.nav.familie.integrasjoner.dokarkiv.metadata
+
+import no.nav.familie.integrasjoner.dokarkiv.client.domene.JournalpostType
+import org.springframework.stereotype.Component
+
+@Component
+object BarnetrygdVedtakMetadata : DokumentMetadata {
```

Comment:
To point out some of the functional stuff I would like to see in ba-sak f.exp. These fields should be set by the client calling this api and not hardcoded in this app.